projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/intro/thruul.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "大师熔炼术",
+  "text": [
+    {
+      "text": "看来你已经进步到这里了工匠，也许你比我想的更聪明。是时候让你的初级熔化炉和那些初级金属变得更强大了！"
+    },
+    {
+      "text": "我希望你已经准备好了，工匠，接下来事情会发生得很快。",	
```

```suggestion
      "text": "我希望你已经准备好了，工匠，接下来我们的进展会逐渐加快了。",	
```
事情会发生的很快？？原意应该是“**很快**（马上）就会**很advanced**”。